kit/brain_board/index.md

```diff
@@ -78,3 +78,19 @@ If you choose to use a tool other than Etcher, you may need to extract the `srob
    Your computer may complain that the SD card is no longer readable, however
    this is expected as the data being written to the SD card is not in a format
    that either Windows or macOS can natively understand.
+
+## Robot Settings
+
+Some of the features on your robot are configured using a settings file, called `robot-settings.toml`. This file is automatically created on your USB drive the first time that you run any code on your robot.
+
+You can edit the settings file using your IDE or any text editor.
```

Comment:
Question: Kinda unrelated to this PR, but does the file use CRLF endings? Wondering how easy the file will be to edit on Windows.

Reply:
Windows Notepad supports Unix line endings since Windows 10 21H1.

Reply:
Oh good point actually... I suspect it generates without them but will read the file back fine if they are present. Opened issue on Astoria to track